src/main/flight/gps_rescue.c

```diff
@@ -97,6 +97,16 @@ typedef struct {
     float accMagnitudeAvg;
 } rescueSensorData_s;
 
+typedef struct {
+    uint32_t previousTimeUs;
+    int8_t gsI; // Stalled movement detection
```

Comment:
Even better than adding a comment about what these do would be to rename them to have names that indicate their use.

Reply:
I've already amended my PR to have proper names in those variables.

Reply:
Amended commit: https://github.com/TonyBlit/betaflight/commit/d2827bae20d677281fcba3bcaff435c38fa009ae